chainbridge/src/mock.rs

```diff
@@ -173,25 +182,27 @@ impl pallet_balances::Config for MockRuntime {
     type Event = Event;
     type ExistentialDeposit = ExistentialDeposit;
     type AccountStore = System;
-    type WeightInfo = ();
     type MaxLocks = ();
+    type MaxReserves = ();
+    type ReserveIdentifier = ();
+    type WeightInfo = ();
 }
 
 // Parameterize chainbridge pallet
 parameter_types! {
     pub const MockChainId: ChainId = 5;
-    pub const ChainBridgePalletId: PalletId = PalletId(*b"chnbridg");
+    pub const ChainbridgePalletId: PalletId = PalletId(*b"cb/bridg");
     pub const ProposalLifetime: u64 = 10;
     pub const RelayerVoteThreshold: u32 = DEFAULT_RELAYER_VOTE_THRESHOLD;
 }
 
 // Implement chainbridge pallet configuration trait for the mock runtime
-impl pallet_chainbridge::Config for MockRuntime {
+impl ChainbridgePalletConfig for MockRuntime {
     type Event = Event;
     type Proposal = Call;
     type ChainId = MockChainId;
-    type PalletId = ChainBridgePalletId;
-    type AdminOrigin = EnsureSignedBy<TestUserId, u64>;
+    type PalletId = ChainbridgePalletId;
```

Comment:
Can we add a constant function that converts the PalletId into an accountId so it can be displayed in the constant section in the api/ui in case it doesnt exist already? It is very convenient to have that.